#3. Write a method which throws exception, Call that method in main class without try block 
def divide():
    raise ZeroDivisionError("Cannot divide by zero.")

divide()  
